pages/api/interview.ts

```diff
@@ -0,0 +1,69 @@
+import { ChatGPTMessage } from "@/constant/constant";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
```

Comment:
Do we need to call this handler everytime and route from there? or we can call the other functions directly?

Reply:
yes, when API is hit, this handler function triggers everytime.